src/Params/ADnoteParameters.cpp

```diff
@@ -175,8 +175,8 @@ static const Ports voicePorts = {
 
 
     //Modulator Stuff
-    rOption(PFMEnabled, rShort("mode"), rOptions(none, mix, ring, phase,
-                frequency, pulse), rLinear(0,127), rDefault(none), "Modulator mode"),
+    rOption(PFMEnabled, rShort("mode"), rOptions(none, mix, ring, am, PM,
+                FM, PWM), rLinear(0,127), rDefault(none), "Modulator mode"),
```

Comment:
This somehow does not match the FLTK items, right? I would expect something like `none, mix, ring, PM, FM, PWM, FMAA-SYNC`.

Reply:
> This somehow does not match the FLTK items, right? I would expect something like `none, mix, ring, PM, FM, PWM, FMAA-SYNC`.

no, 
the sync is enabled with PsyncEnabled. it can be combined with every modulation mode
the AA is enabled with the existing AAenabled port.
only new thing is AM mode which is a variation of ring